client/src/app/api/models.ts

```diff
@@ -604,17 +608,26 @@ export type AggregateTicketStatus =
   | "In Progress"
   | "Completed"
   | "Error"
-  | "No Issues";
+  | "No Issues"
+  | "Not Started";
```

Comment:
Not sure where that one comes from.
I can't see that in the requirements.
I seems redundant with "Issues created"

Reply:
![image](https://github.com/konveyor/tackle2-ui/assets/11218376/eeb5ccbb-6d00-41d6-b733-800374dcfaa4)
Grabbed it from the enhancement

Reply:
https://github.com/konveyor/enhancements/tree/master/enhancements/tackle/tackle-jira-integration